go.mod

```diff
@@ -263,8 +265,6 @@ replace (
 	// in order to try and avoid consensus hash mismatches.
 	// See: https://github.com/palomachain/paloma/issues/1144
 	// See https://github.com/palomachain/paloma/pull/1139
-	cosmossdk.io/store v1.0.2 => cosmossdk.io/store v1.1.0
-	github.com/cosmos/iavl => github.com/cosmos/iavl v1.1.2
-	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.18.0
-	github.com/prometheus/common => github.com/prometheus/common v0.47.0
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.19.0
+	github.com/prometheus/common => github.com/prometheus/common v0.52.2
```

Comment:
These can both be removed I think, Cosmos `0.50.6` pulls them in as well.
Meaning we can hopefully remove the entire `replace` block here.

Reply:
resolving this.